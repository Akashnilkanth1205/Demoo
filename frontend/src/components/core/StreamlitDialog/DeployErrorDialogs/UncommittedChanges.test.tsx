import UncommittedChanges from "./UncommittedChanges"

describe("DeployErrorDialog - RepoIsAhead", () => {
  it("Should render without crashing", () => {
    const dialog = UncommittedChanges("user/reponame")

    expect(dialog).toMatchSnapshot()
  })
})
