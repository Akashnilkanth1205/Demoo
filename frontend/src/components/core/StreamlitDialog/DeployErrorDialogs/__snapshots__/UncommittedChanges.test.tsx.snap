// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DeployErrorDialog - RepoIsAhead Should render without crashing 1`] = `
Object {
  "body": <React.Fragment>
    <p>
      The file 
      <strong>
        module
      </strong>
       has uncommitted changes.
    </p>
    <p>
      Please commit the latest changes and push to Github to continue.
    </p>
  </React.Fragment>,
  "title": "Error deploying app",
}
`;
